legate/driver/main.py

```diff
@@ -17,32 +17,26 @@
 """
 from __future__ import annotations
 
-__all__ = ("main",)
+from typing import Type, Union
 
+from . import CanonicalDriver, LegateDriver
 
-def main(argv: list[str]) -> int:
-    """A main function for the Legate driver that can be used programmatically
-    or by entry-points.
+__all__ = "legate_main"
```

Comment:
value for all needs to be a sequence
```suggestion
__all__ = ("legate_main",)
```